lib/iris/cube.py

```diff
@@ -1611,10 +1611,10 @@ def core_data(self):
         to be decided: should this be public??
 
         """
-        if self._numpy_array is not None:
```

Comment:
We've defined a method to make this check, so let's use it rather than repeating the same code.